tests/webTables.spec.ts

```diff
@@ -237,25 +233,25 @@ test('Validate specialty lists', async ({ page }) => {
     
     /* 5. On the Veterinarians page, locate the "Sharon Jenkins" in the list and click "Edit" button */
     // Locate Sharon Jenkins and click 'Edit'
-    await page.getByRole('row', {name: 'Sharon Jenkins'}).getByRole('cell').last().getByRole('button', {name: 'Edit Vet'}).click()
+    await page.getByRole('row', {name: 'Sharon Jenkins'}).getByRole('cell').last().getByText('Edit Vet').click()
```

Comment:
How about cell? :)
You have changed the method type to get by text, which is fine as well, but keeping not needed "cell" I have mentioned in my comment )))